js/src/modal/modal.js

```diff
@@ -432,8 +439,20 @@ class Modal {
   }
 
   _checkScrollbar() {
-    const rect = document.body.getBoundingClientRect()
-    this._isBodyOverflowing = rect.left + rect.right < window.innerWidth
+    let contentHeight = null
+
+    if (!bodyStyle) {
+      bodyStyle = window.getComputedStyle(document.body)
```

Comment:
bodyStyle can change over the time by doing that you freeze it, so you should remove the global var and query the body style each time it's needed

Reply:
it's a possible case, but user can change body styles too